Jenkinsfile

```diff
@@ -7,6 +7,9 @@ node {
       checkout scm
     }
 
+    env.RF_SETTINGS_BUCKET = 'rasterfoundry-staging-config-us-east-1'
+    env.RF_AWS_CONFIG_BUCKET = 'rasterfoundry-testing-config-us-east-1'
+    env.RF_AWS_ARTIFACTS_BUCKET = 'rasterfoundry-global-artifacts-us-east-1'
```

Comment:
Can we introduce a newline here before the comment? Also, it may make sense to move the `AWS_DEFAULT_REGION` up here as well.